src/plots/linePlot.ts

```diff
@@ -425,6 +450,74 @@ export module Plots {
       return filteredDataIndices;
     }
 
+    private _filterDownsampling(dataset: Dataset, indices: number[]) {
```

Comment:
why do you need 2 functions here? it seems like _filterDownsampling doesn't do much other than getting accessors